pkg/config/config.go

```diff
@@ -55,6 +55,16 @@ type Config struct {
 		Variables               map[string]string `toml:",omitempty"`
 		Registries              []Registry        `toml:",omitempty"`
 	} `toml:",omitempty"`
+
+	Sso struct {
+		// add any global configuration to sso profile commands defaults here.
+		StartURL            string
+		SsoRegion           string
```

Comment:
nit: we prefer to capitalise acronyms in our Go code, so 'SsoRegion' would be 'SSORegion'